src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -1,26 +1,14 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
```

Comment:
Do we care about keeping the extra usings?

Reply:
Not really.  Logging fits in this scenario so, it isn't entirely unnecessary.